### ISSUE TITLE
[A11Y] No text alternative for the header logo

### ISSUE THREAD

**Issue by foxbunny** [indico/indico#5934]
**Describe the bug**

The header logo does not have a text alternative, and therefore the link does not have a label.

**Additional context**

- https://www.w3.org/WAI/WCAG21/Understanding/non-text-content.html
- https://talk.getindico.io/t/non-text-content-logo/3201
